package.json

```diff
@@ -69,8 +69,8 @@
     "react": "^16.8.6",
     "react-addons-pure-render-mixin": "^15.6.2",
     "react-dom": "^16.8.6",
-    "react-with-styles-interface-aphrodite": "^5.0.1",
-    "react-with-styles-interface-css-compiler": "^1.1.2",
+    "react-with-styles-interface-aphrodite": "^6.0.0",
+    "react-with-styles-interface-css-compiler": "^2.1.0",
```

Comment:
- `react-with-styles-interface-aphrodite@6` includes react-with-styles@4 in peer deps
- `react-with-styles-interface-css-compiler@2.1.0` includes react-with-styles-interface-css@5, which uses `react-with-styles@4`